drivers/soundwire/cadence_master.c

```diff
@@ -868,6 +869,20 @@ int sdw_cdns_enable_interrupt(struct sdw_cdns *cdns, bool state)
 		mask = interrupt_mask;
 
 update_masks:
+	/* clear slave interrupt status before enabling interrupt */
```

Comment:
typos in commit message

soundwire: cadence_master: clear slave interrupt status before enabling interrupt

make sure all interrupts status are cleared before enabling interrupt
so that there is no unexpected interrupt triggered.

Reply:
..., never ignore check tools